app-dicts/myspell-nl/myspell-nl-2.10g-r2.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
This ebuild is already EAPI-7, so this commit seems pointless.

Reply:
Dropped